extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -236,6 +244,24 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     super.onCreativeRender(isVerifiedAmpCreative);
     injectActiveViewAmpAnalyticsElement(
       this, this.extensions_, this.ampAnalyticsConfig);
+    this.centerCreative_();
+  }
+
+  /**
+   * Centers the outermost creative iframe inside the ad slot.
+   * @private
+   */
+  centerCreative_() {
+    const iframe = this.element.getElementsByTagName('iframe')[0];
+    if (iframe) {
+      setStyles(iframe, {
+        top: '50%',
+        left: '50%',
+        bottom: '',
+        right: '',
+        transform: 'translate(-50%, -50%)',
```

Comment:
Also, look into `src/base-element.js#applyFillContent`.  It's called from `renderAmpCreative_` and sets some style classes.  Those might have some things to say about centering -- I haven't looked.  There's also a `SHARED_IFRAME_PROPERTIES` that sets up properties on all cross-domain iframes -- that might be a better place to put centering styling?